src/frontend/src/pages/CreateWorkPackagePage/CreateWorkPackageForm.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { useHistory } from 'react-router-dom';
+import { isProject, validateWBS } from 'shared';
+import { useAuth } from '../../hooks/auth.hooks';
+import { useCreateSingleWorkPackage } from '../../hooks/work-packages.hooks';
+import { routes } from '../../utils/routes';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import CreateWorkPackageFormView from './CreateWorkPackageFormView';
+
+export interface CreateWorkPackageFormInputs {
+  name: string;
+  startDate: Date;
+  duration: number;
+  crId: string;
+  wbsNum: string;
+  dependencies: { wbsNum: string }[];
+  expectedActivities: { bulletId: number; detail: string }[];
+  deliverables: { bulletId: number; detail: string }[];
+}
+
+const CreateWorkPackageForm: React.FC = () => {
+  const history = useHistory();
+  const auth = useAuth();
+
+  const { isLoading, mutateAsync } = useCreateSingleWorkPackage();
+
+  if (isLoading || auth.user === undefined) return <LoadingIndicator />;
```

Comment:
so to do what we want we can do

```
if (isLoading || !auth.user) return <LoadingIndicator />;
const { userId } = auth.user;

const handleSubmit = blah blah blah {
   ...
    try {
        await mutateAsync({userId
    }
}
```


BUT WAIT i acutally just remembered that because of refactoring and auth stuff you don't need to send the userId at all so you can skip most of this